clients/js/src/leafAssetId.ts

```diff
@@ -55,7 +55,7 @@ export async function parseLeafFromMintToCollectionV1Transaction(
 
```

Comment:
So its worth noting this will be a JS client breaking change for all who use the helper.

Reply:
Yeah. I should make a ticket to make this smarter now that we have the TX parser in umi.